Makefile

```diff
@@ -1,6 +1,8 @@
 SHELL=/bin/bash
 export PATH := ./bin:./venv/bin:$(PATH)
 
+PYTHON ?= $(shell which python3.5 || which python3.6)
```

Comment:
here, we detect which python to use. the `?=` assignment allows `PYTHON` to be overriden via `make PYTHON=whatever-you-want`, to prevent needing to update this in the future

note that it still is a minimum of python 3.5